package.json

```diff
@@ -130,6 +130,7 @@
     "pkg-dir": "^5.0.0",
     "reflect-metadata": "^0.1.13",
     "tsconfig-paths": "^3.11.0",
+    "tslib": "^2.0.0",
```

Comment:
We already have `"tslib": "^2.3.1",` in devDependencies, is it ok to just copy that here (and remove from devDeps)?